<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TensorFlow Testbed</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Input image</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string>MNIST image</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>131</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Model Builder</string>
    </property>
    <widget class="QPushButton" name="TrainModel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>71</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Train model</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Evaluation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>151</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Evaluation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Prediction">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Prediction</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BuildModel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Build model</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoadModel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load model</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
